modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating a cluster to a heterogenous payload using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* You must be updated to {product-title} version 4.13.0 or higher. For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_ listed in the Additional resources section.
+
+.Procedure
+
+. Install your {product-title} cluster on your preferred platform. For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_ in the Additional resources section.
```

Comment:
>Install your {product-title} cluster on your preferred platform

Currently multi-arch is supported on aws and azure only.

Reply:
yes, confirmed. And it will only be support for connected environments.

Reply:
I'd also say that the first point (Install your... cluster...) should stay in the pre-requisites. This procedure is to migrate and existing single-arch-payload-based cluster to a multi-arch payload. Therefore, I'd expect users of this are users having single-arch clusters already installed and that want to perform this day2 operation to later add workers of a secondary architecture to their cluster. Wdyt @shellyyang1989 ?

Reply:
Good point! Right, we should mention it in pre-requisites.